integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -378,12 +378,10 @@ public static boolean scaleCluster(String domainUid, String namespace, String cl
       }
     }
 
-    // TODO: Below needs to be modifed for v9
     // construct the patch string for scaling the cluster in the domain
     StringBuffer patchStr = new StringBuffer("[{")
         .append("\"op\": \"replace\", ")
-        .append("\"path\": \"/spec/clusters/")
-        .append(index)
+        .append("\"path\": \"/spec")
```

Comment:
As this util method is scaling a particular cluster, its better to patch the cluster resource instead of domain resource. In any case, we need to cover scaling by patching domain resource and cluster resource.

Reply:
Patching cluster resource will be separate utility. This method belongs to Domain level utility.

Reply:
Actually this needs to be moved to Cluster utils as its scaling a particular cluster. Domain level scale method will scale all the clusters.

Reply:
Add a new  method scaleClusters() to utils/DomainUtils